### PR TITLE
Translations: grenades

### PR DESCRIPTION
**Pull request by Emojigit**

### CODE REVIEW

File: mods/pvp/grenades/grenades.lua [hunk 1]
```diff
@@ -122,7 +126,8 @@ grenades.register_grenade("grenades:frag_sticky", fragdef_sticky)
 local sounds = {}
 local SMOKE_GRENADE_TIME = 30
 grenades.register_grenade("grenades:smoke", {
-	description = "Smoke grenade (Generates smoke around blast site)",
+	description = S("Smoke grenade (Generates smoke around blast site)"),
+    short_description = S("Smoke grenade"),
```
Comment (LoneWolfHT) [inline mods/pvp/grenades/grenades.lua:130]: The indentation here looks wrong.

I don't you need to add a `short_description`, this mod isn't going anywhere else

Comment (Emojigit) [reply mods/pvp/grenades/grenades.lua:130]: So, remove the short description, right?

Comment (LoneWolfHT) [reply mods/pvp/grenades/grenades.lua:130]: Yeah

**Comment by savilli** [comment]
@Emojigit Are you planning to work on it?

**Comment by farooqkz** [comment]
ping @Emojigit

**Comment by Emojigit** [comment]
Closing this. I will rework on translations later.

**Pull request closed without merging**